docs/deployment.md

```diff
@@ -69,29 +94,24 @@ After deploying Standalone TripleO, there are a few post-deployment configuratio
 
 ### Ironic Policy
 
-The Ironic policy file must be updated if you intend for non-admins to use the Ironic API. Access
-for single-node API fuctions is granted through the use of the ``is_node_owner`` role, which applies
-to a project specified by a node's ``owner`` field. Access to ``baremetal:node:list`` can safely be
-opened to all, as that API call will filter results for non-admins by owner. Access to
-``baremetal:node:create`` should not be exposed to non-admins.
+The Ironic policy file must be updated if you intend for non-admins to use the Ironic API. Access for single-node API fuctions is granted through the use of the ``is_node_owner`` and ``is_node_lessee` roles, which apply to a project specified by a node's ``owner`` and ``lessee`` field respectively. Further detail can be found in the upstream [node multi-tenancy documentation](https://docs.openstack.org/ironic/latest/admin/node-multitenancy.html)
+
+This repository includes a sample Ironic policy file that enables node owners to manage their nodes and node lessees to provision their leased nodes. It can be found at [/etc/ironic/policy.json.sample](/etc/ironic/policy.json.sample).
+
+### Neutron Policy
 
-For example, if you would like to update the default Ironic policy settings to allow non-admins to be
-able to list/get/set the power state of nodes that they own, update the Ironic policy file as follows:
+If you are using ansible networking to configure the switch, the following Neutron policy rule must be updated to allow non-admins to attach VLAN networks to their nodes:
 
 ```
-# Retrieve a single Node record
-# GET  /nodes/{node_ident}
-#"baremetal:node:get": "rule:is_admin or rule:is_observer"
-"baremetal:node:get": "rule:is_admin or rule:is_observer or role:is_node_owner"
-
-# Retrieve multiple Node records, filtered by owner
-# GET  /nodes
-# GET  /nodes/detail
-#"baremetal:node:list": "rule:baremetal:node:get"
-"baremetal:node:list": ""
-
-# Change Node power status
-# PUT  /nodes/{node_ident}/states/power
-#"baremetal:node:set_power_state": "rule:is_admin"
-"baremetal:node:set_power_state": "rule:is_admin or rule:is_node_owner"
-```
\ No newline at end of file
+# Get ``provider:physical_network`` attribute of a network
+# GET  /networks
+# GET  /networks/{id}
+#"get_network:provider:physical_network": "rule:admin_only"
+"get_network:provider:physical_network": "rule:admin_or_owner or rule:shared"
+```
+
+### Ansible Networking and Cisco Nexus Switches
+
+If you are using a Cisco Nexus switch, then you'll need an updated Ansible playbook for configuring trunk ports. It can be found at https://github.com/ansible-network/network-runner/blob/devel/etc/ansible/roles/network-runner/providers/nxos/conf_trunk_port.yaml.
+
+In addition, if you require PortFast mode, then you'll need to update the access port and trunk port Ansible playbooks to include PortFast configuration commands.
```

Comment:
Is it going to be obvious to people what updates are necessary?

Reply:
I haven't actually tested the needed command (assuming that the upstream solution will come along soon enough), but I've added some text around this.